### PR TITLE
Adjust target size of interactive elements to meet accessibility criteria

### PR DESCRIPTION
**Pull request by epetrow**
Part of: https://github.com/telerik/web-components-planning/issues/125
Closes: https://github.com/telerik/kendo-themes-private/issues/40

### CODE REVIEW

File: packages/fluent/scss/splitter/_variables.scss [hunk 1]
```diff
@@ -37,6 +37,13 @@ $kendo-splitter-drag-handle-thickness: map.get( $kendo-spacing, 0.5 ) !default;
 /// @group splitter
 $kendo-splitter-drag-icon-margin: map.get( $kendo-spacing, 2 ) !default;
 
+/// Horizontal padding of the  collapse icon in the splitter.
```
Comment (inikolova) [inline packages/fluent/scss/splitter/_variables.scss:40]: Remove the double spacing before the "collapse icon" in the comment

File: packages/fluent/scss/splitter/_variables.scss [hunk 2]
```diff
@@ -37,6 +37,13 @@ $kendo-splitter-drag-handle-thickness: map.get( $kendo-spacing, 0.5 ) !default;
...
+/// @group splitter
+$kendo-splitter-collapse-icon-padding-x: null !default;
+/// Vertical padding of the  collapse icon in the splitter.
```
Comment (inikolova) [inline packages/fluent/scss/splitter/_variables.scss:43]: Same as the above

**Comment by inikolova** [comment]
I observed the following misalignments - maybe some of them are expected, however could we double check :)

- Icons and checkboxes in the following tests ->
   -- multiselecttree/multiselecttree-opened -> https://github.com/telerik/kendo-themes/pull/4488/commits/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-cf9889a10ff476fdf127627d1ec356114038aa94c18822b18df7fa58536454ce
   -- dropdowntree/dropdowntree-opened/ -> https://github.com/telerik/kendo-themes/pull/4488/commits/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-7adb1fb8293882db715a2206e1dd952d3ca025d3af96bf792b3412499d9f7303
   -- pivotgrid/pivotgrid-column-menus.png -> https://github.com/telerik/kendo-themes/pull/4488/commits/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-525779bfb1e02c7eb0388691c497e9d28b0450bf2f952d39a4fcf31489a37d45

- Maybe we could move the whole content to the right, as the icon is too close to the border: 
    -- spreadsheet/spreadsheet-filter-menu.png -> https://github.com/telerik/kendo-themes/pull/4488/commits/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-4e7390ecff577542d0fc97a6c18d32c065eeb82de9e9b01287de87b19e6f471b

**Comment by epetrow** [comment]
> I observed the following misalignments - maybe some of them are expected, however could we double check :)
> 
> * Icons and checkboxes in the following tests ->
>   -- multiselecttree/multiselecttree-opened -> [f169c5f#diff-cf9889a10ff476fdf127627d1ec356114038aa94c18822b18df7fa58536454ce](https://github.com/telerik/kendo-themes/commit/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-cf9889a10ff476fdf127627d1ec356114038aa94c18822b18df7fa58536454ce)
>   -- dropdowntree/dropdowntree-opened/ -> [f169c5f#diff-7adb1fb8293882db715a2206e1dd952d3ca025d3af96bf792b3412499d9f7303](https://github.com/telerik/kendo-themes/commit/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-7adb1fb8293882db715a2206e1dd952d3ca025d3af96bf792b3412499d9f7303)
>   -- pivotgrid/pivotgrid-column-menus.png -> [f169c5f#diff-525779bfb1e02c7eb0388691c497e9d28b0450bf2f952d39a4fcf31489a37d45](https://github.com/telerik/kendo-themes/commit/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-525779bfb1e02c7eb0388691c497e9d28b0450bf2f952d39a4fcf31489a37d45)
> * Maybe we could move the whole content to the right, as the icon is too close to the border:
>   -- spreadsheet/spreadsheet-filter-menu.png -> [f169c5f#diff-4e7390ecff577542d0fc97a6c18d32c065eeb82de9e9b01287de87b19e6f471b](https://github.com/telerik/kendo-themes/commit/f169c5f43d3ca832d13d16d0c81d95e00d66f58f#diff-4e7390ecff577542d0fc97a6c18d32c065eeb82de9e9b01287de87b19e6f471b)

I synced with @dmanova and agreed on the following:

 - DropDownTree, MultiSelectTree - we will add additional inline padding to the TreeView inside the popup ( we were already overriding it's padding in the `_layout` of the DropDownTree )
 - PivotGrid - the changes are expected, no changes need to be applied for this case
 - Spreadsheet's filter menu - same as the first case, since we were already modifying the padding of the TreeView, additional inline padding will be added

**Review by inikolova — APPROVED** [review]

**Review by Juveniel — APPROVED** [review]

**Pull request merged**